<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to wave</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Table layout</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="layoutSimple">
        <property name="text">
         <string>Simple (8192 samples)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layoutButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="firstSampleChk">
        <property name="text">
         <string>First sample only</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="layoutSerum">
        <property name="text">
         <string>Serum® like (64×2048 samples)</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layoutButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="serumWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Interpolation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="serumCombo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>None (repeated values)</string>
            </property>
            <property name="icon">
             <iconset theme="labplot-xy-smoothing-curve">
              <normaloff/>
             </iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wave based</string>
            </property>
            <property name="icon">
             <iconset theme="labplot-xy-interpolation-curve">
              <normaloff/>
             </iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Table based</string>
            </property>
            <property name="icon">
             <iconset theme="labplot-xy-equation-curve">
              <normaloff/>
             </iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="serumWhatLbl"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Audio format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bit depth</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="bitCombo">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>16bit integer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24bit integer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32bit integer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32bit float</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="s16i">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QRadioButton" name="s16iRound">
            <property name="text">
             <string>Round</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">s16iButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="s16iDecimateMax">
            <property name="text">
             <string>Decimate max</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">s16iButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="clipLbl">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>(no clipping detected)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Value conversion</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="s24i">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Value conversion</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="s24iRound">
            <property name="text">
             <string>Round up</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">s24iButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="s24iKeep">
            <property name="text">
             <string>Original values</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">s24iButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="s32page">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>No conversion options
(values will be scaled)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layoutSerum</sender>
   <signal>toggled(bool)</signal>
   <receiver>serumCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="layoutButtonGroup"/>
  <buttongroup name="s16iButtonGroup"/>
  <buttongroup name="s24iButtonGroup"/>
 </buttongroups>
</ui>
